index.html

```diff
@@ -18,15 +18,130 @@
     <section class="intro">
         <h1>Hey There. I'm David</h1>
         <h2>I'm a Software Developer</h2>
-        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
             don’t hestiate to contact me.</p>
         <ul class="social-links">
             <li><a href="#"><img src="./images/github.svg" alt="GitHub"></a></li>
-            <li><a href="#"><img src="./images/linked-in.svg" alt="Linked In"></a></li>
+            <li><a href="#"><img src="./images/linked-in.svg" alt="LinkedIn"></a></li>
             <li><a href="#"><img src="./images/peace.svg" alt="Peace"></a></li>
             <li><a href="#"><img src="./images/twitter.svg" alt="Twitter"></a></li>
             <li><a href="#"><img src="./images/m.svg" alt="M"></a></li>
         </ul>
     </section>
+    <main>
+      <h1>My Recent Works</h1>
```

Comment:
## Optional
`<h1>` defines the most important heading. and it is used only once on the page. You can change it with h2

- [ ]  Please open this link to learn more about h1 to h6 tag: [h1 to h6 tags](https://www.w3schools.com/tags/tag_hn.asp)